src/csharp/Microsoft.Spark.E2ETest/IpcTests/BroadcastTests.cs

```diff
@@ -32,12 +32,36 @@ public BroadcastTests(SparkFixture fixture)
             _df = _spark.CreateDataFrame(new[] { "hello", "world" });
         }
 
+        /// <summary>
+        /// Test Broadcast support by using multiple broadcast variables in a UDF with
+        /// encryption enabled.
+        /// </summary>
+        [SkipIfSparkVersionIsLessThan(Versions.V2_3_2)]
+        public void TestMultipleBroadcastWithEncryption()
```

Comment:
Can we reuse a lot of these tests?  Just pass in some parameters to the Test methods to test with encryption enabled/disabled.  You can use `Theory` and `Inline`

Reply:
I am not sure how to do that given how encryption wasn't supported before Spark 2.3.2. In order to be able to skip tests for Spark versions below that, don't we need to have separate tests? Please let me know if there is a way to run/skip tests based on`InlineData`.

Reply:
No it does not break, but we will basically be testing the same logic multiple times. If that is ok, then I will combine `TestMultipleBroadcastWithoutEncryption` and `TestMultipleBroadcastWithEncryption` into `TestMultipleBroadcast`, and leave the remaining 3 as is? Please let me know if that sounds ok

Reply:
That sounds good.  I think most of these tests with encryption on/off.

Reply:
Done. Thanks!